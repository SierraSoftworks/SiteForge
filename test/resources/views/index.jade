extends layout

block body
	h1 SiteForge
	p.
		Welcome to SiteForge, a custom site generator built with Node.js, Jade and Less.