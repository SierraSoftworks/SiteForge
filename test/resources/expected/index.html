<!DOCTYPE html><html><head><title>SiteForge</title><link rel="stylesheet" href="/style.min.css"></head><body><h1>SiteForge</h1><p>Welcome to SiteForge, a custom site generator built with Node.js, Jade and Less.</p></body></html>